src/App.css

```diff
@@ -294,7 +294,7 @@ a.button-variable-width {
   min-width: 0;
 }
 
-a.button[disabled] {
+a.button.disabled {
```

Comment:
`disabled` is not a valid attribute of an anchor tag (the TypeScript compiler complains), so I've switched to using a class name here.